frontend/src/components/ui-kit/components/Button/Button.module.scss

```diff
@@ -34,10 +35,19 @@
   padding: 12px 25px;
 }
 .padding-sm {
-  padding: 12px 12px 12px 25px;
+  padding: 12px 14px 12px 23px;
```

Comment:
Why did the paddings change? This applies to all paddings

Reply:
There were changes in the design